benchpress/install_oss_performance.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Installs HHVM's oss-performance under benchmarks/
+#
+
+HHVM_PATH=${HHVM_PATH:-'hhvm'}
```

Comment:
Does this assume hhvm is installed or we are installing it using benchpress before calling the oss-performance installer?